constructors/library/test/bezierspline_test/out/bezierIn.json

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
Maybe the test output can add noise to the repo. Are they being used in CI or somewhere?

Reply:
yes, these tests contain fixtures and they normally pass the in/ to the function and compare that the out/ matches what is expected